rspec-tools/rspec_tools/coverage.py

```diff
@@ -197,9 +202,13 @@ def is_version_tag(name):
 
 
 def comparable_version(key):
-  if not is_version_tag(key):
-    return [0]
-  return list(map(int, key.split('.')))
+  v = key.removeprefix('sqcb-').removeprefix('sqs-')
+  if not is_version_tag(v):
+    if v == 'master':
+      return [0]
+    else:
+      sys.exit(f'Unexpected version {key}')
+  return list(map(int, v.split('.')))
```

Comment:
Let's make the flow more linear between condition and action
```suggestion
  if is_version_tag(v):
    return list(map(int, v.split('.')))
  if v == 'master':
    return [0]
  sys.exit(f'Unexpected version {key}')
  
```

Reply:
f5d9bbf6ab